lib/fedora_migrate/rels_ext_datastream_mover.rb

```diff
@@ -4,26 +4,32 @@ class RelsExtDatastreamMover < Mover
     RELS_EXT_DATASTREAM = "RELS-EXT".freeze
 
     def migrate
-      statements.each do |statement|
-        subject.ldp_source.graph << [subject.rdf_subject, migrate_predicate(statement.predicate), migrate_object(statement.object)]
-      end
-      subject.ldp_source.update
+      migrate_statements
+      target.ldp_source.update
       update_index
     end
 
+    def post_initialize
+      @target ||= ActiveFedora::Base.find(source.pid.split(/:/).last)
```

Comment:
RE: target, it's always an attr_accessor but sometimes you have to use @target to indicate it's the accessor and not a plain variable.
RE: id_component, no, because that method is using a URI, but the same thought struck me because I use the same logic in other classes.